README.md

```diff
@@ -1,5 +1,22 @@
-# CDV_Muschel-Mosaik
-Muscheln und Schnecken des Museums für Naturkunde Berlin freistellen und neue Bilder generieren. Erweiterung, dass man ein Selfie von sich nutzen kann und Metainfos durch Antippen auslesen kann.
+# mollex: mollusc extractor
+
+## English
+mollex extracts molluscoid shapes from images.
```

Comment:
what's up with the weird line breaks? are they for readability of the source code? they won't show in the rendered readme, since you're missing the two spaces at the end of the line.

Reply:
Yes, they are for readability. Adding line breaks at convenient locations is a habit I picked up writing LaTeX - it simplifies adding text later on since the beginning and end of a line are very likely to be at the position you want to insert text.